lib/index.js

```diff
@@ -0,0 +1,2 @@
+require("babel/register");
```

Comment:
Babel hooks into global require which means it will parse not only your files but the files you require in from external libraries. For both safety and performance you may want to consider renaming your js files (`.es6.js`?) and only registering what you need.